src/templates/home/home.html

```diff
@@ -25,26 +25,44 @@ <h1>Collectivités, trouvez en quelques clics les aides pertinentes pour
     </div>
 </section>
 
-<section id="contact">
-    <div class="narrow-form">
-        <h1>Recevoir notre actualité</h1>
+    <section id="partner">
+        <div class="container">
+            <h1>Retrouvez sur aides-territoires les aides de :</h1>
```

Comment:
Attention à bien utiliser des espaces insécables avant le « : », ce qui évite sur certaines tailles d'écrans d'avoir un double-point qui se balade tout seul sur sa ligne.

```suggestion
            <h1>Retrouvez sur aides-territoires les aides de :</h1>
```